scripts/rnnlm/lmrescore_pruned.sh

```diff
@@ -26,6 +26,8 @@ normalize=false # If true, we add a normalization step to the output of the RNNL
                 # as in our RNNLM setup, a properly trained network would automatically
                 # have its normalization term close to 1. The details of this
                 # could be found at http://www.danielpovey.com/files/2018_icassp_rnnlm.pdf
+lattice_prune_beam=4 # Beam used in pruned lattice composition
```

Comment:
@hainan-xv, in your experience, how much difference does this make to WER results?  (4 vs. 6)